alphagov/paas-product-pages
# Regular maintenance

## What

- Regular dependency updates
- Update import of govuk-frontend to accommodate move to EcmaScript Modules

## How to test

- review commits
- clone branch
- install updated dependencies (`npm install`)
- build the site (`npm run build`)
- `cd` into `out` folder and serve the page (`npx serve`)
- check page on `localhost:5000` for any issues

There are [expected warnings in CSS compilation](https://github.com/alphagov/govuk-frontend/issues/2238) coming from govuk-frontend
